### ISSUE TITLE
Into the Breach crash with "Save Screen" enabled.

### ISSUE THREAD

**Issue by R30hedron** [clementgallet/libTAS#364]
Game: Into the Breach (version 1.2.23 (15-May-2020); DRM free)
libTAS version: commit fe7b8dd (14-Aug-2020)
Executable options: None set
Time tracking: `clock_gettime()`

Running Into the Breach in libTAS, with the `Runtime > Save Screen` setting enabled, has a crash on frame 15 as the game is loading. Disabling this setting allows the game to run as expected past this point, though this does mean missing out on a few non-essential features such as screen refreshing on save state loads or previewing inputs during movie recordings.

**Comment by clementgallet** [comment]
The game redirects stderr into the file `~/.local/share/IntoTheBreach/error.txt` with some unknown method (ie not `dup2()`). For now, we need to look into that file for logging.

Relevent log:
```
[libTAS f:0] Thread 60828 (main) SDL_CreateRenderer call.
[libTAS f:0] Thread 60828 (main) XGetWindowAttributes called with window 79691796
[libTAS f:0] Thread 60828 (main) glXMakeCurrent call.
[libTAS f:0] Thread 60828 (main) glXGetSwapIntervalMESA call.
[libTAS f:0] Thread 60828 (main) Inited Screen Capture with dimensions (1280,720)
[libTAS f:0] Thread 60828 (main) SDL_RenderPresent call.
[libTAS f:1] Thread 60828 (main) Generate SDL event SDL_CONTROLLERDEVICEADDED with joy 0
[libTAS f:1] Thread 60828 (main) Generate SDL event SDL_JOYDEVICEADDED with joy 0
[libTAS f:1] Thread 60828 (main) SDL_DestroyRenderer call.
[libTAS f:1] Thread 60828 (main) glXMakeCurrent call.
[libTAS f:1] Thread 60828 (main) SDL_GL_CreateContext call.
[libTAS f:1] Thread 60828 (main) Disable vsync !!
[libTAS f:1] Thread 60828 (main) ERROR: SDL_GetRenderer failed: -135535328
[libTAS f:1] Thread 60828 (main) ERROR: SDL_CreateTexture failed: -135535328
```

It seems the game checks for SDL_Render* rendering API, but then switches back to SDL_GL* rendering. libTAS is probably confused and still wants to use SDL_Render API for capturing the game screen.

**Comment by clementgallet** [comment]
Fixed by 3e4ff00.
